app/main/views.py

```diff
@@ -8,10 +8,18 @@
 from .. import db
 from ..models import EditableHTML, Resource, Rating, Descriptor, OptionAssociation, RequiredOptionDescriptor
 from . import main
+from wtforms.fields import SelectMultipleField, TextAreaField
+from ..single_resource.forms import SingleResourceForm
+
 from datetime import datetime
 
 @main.route('/')
 def index():
+    options = Descriptor.query.all()
```

Comment:
we should also filter out the required descriptor from these so that we don't display a dropdown for it twice